cobbler/cobbler_collections/manager.py

```diff
@@ -180,6 +181,8 @@ def deserialize(self):
 
         :raises CX: if there is an error in deserialization
         """
+        old_cache_enabled = self.api.settings().cache_enabled
```

Comment:
If we have a safeguard during object creation that the cache is not invalidated. Why is this needed? Maybe I am overlooking something but shouldn't this then be superfluous?

Reply:
The execution of the full deserialization code for all collections can only be done when cobbler is started. Therefore, it is safe, but architecturally incorrect. Unnecessary attempts to clear the cache makes the the cobbler's start 10 times slower. I inserted this code as a temporary one until the deserialization code that takes into account object relationships is ready.